src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Participation.java

```diff
@@ -0,0 +1,56 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import javax.persistence.*;
+
+@Entity
+public class Participation {
+
+  @Id @GeneratedValue private Long id;
+
+  private boolean active;
+
+  private Integer participationNumber;
+
+  @ManyToOne(cascade = {CascadeType.PERSIST})
+  private Game game;
+
+  @ManyToOne private Player player;
+
+  public Long getId() {
+    return id;
+  }
+
+  @JsonIgnore
+  public void setId(Long id) {
```

Comment:
use @JsonIgnore that the swagger ui does not show the id as writeable.
(its ignored anyway)